airflow/cli/commands/celery_command.py

```diff
@@ -96,6 +100,27 @@ def _serve_logs(skip_serve_logs: bool = False):
         sub_proc.terminate()
 
 
+@after_setup_logger.connect()
+def logger_setup_handler(logger, **kwargs):
+    # Setup levels at which logs go to stderr and stdout if required
+    if conf.get("logging", "celery_logging_split", fallback=None):
+        celery_formatter = logging.Formatter(DEFAULT_TASK_LOG_FMT)
+
+        class NoErrorOrAboveFilter(logging.Filter):
+            def filter(self, record):
+                return record.levelno <= logging.WARNING
+
+        below_error_handler = logging.StreamHandler(sys.stdout)
+        below_error_handler.addFilter(NoErrorOrAboveFilter())
```

Comment:
How is this different from `setLevel`?

Reply:
I see, I misunderstood. Perhaps a docstring in `NoErrorOrAboveFilter` could help prevent the same error by others?

Reply:
Great point. I will suggest one ASAP.

Reply:
@uranusjr what do you think now?

Reply:
I flipped the wording to a positive tone (I think it avoids some mental overhead)